torchtune/modules/loss/loss_types.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from abc import ABC, abstractmethod
+
+import torch
+from torch import nn
+
+
+class SFTLoss(ABC):
+    """Interface for loss functions in torchtune used in sft recipes."""
+
+    def apply_compile_strategy(self, *args, **kwargs):
+        """Compile the loss function for inference."""
+        self.forward = torch.compile(self.forward, *args, **kwargs)
+        return self
```

Comment:
Why bother defining this here? We seem to define it in the child classes anyways

Reply:
hmm ok iNheRItAnCe